OpenMEEG/libs/OpenMEEG/src/assembleSourceMat.cpp

```diff
@@ -153,31 +156,35 @@ namespace OpenMEEG {
         mat = Matrix((geo.size()-geo.nb_current_barrier_triangles()), n_sensors);
         mat.set(0.0);
 
-        for (Geometry::const_iterator mit0=geo.begin();mit0!=geo.end();++mit0){
+        for (Geometry::const_iterator mit0 = geo.begin(); mit0 != geo.end(); ++mit0) {
             if (mit0->current_barrier()) {
-                for (Geometry::const_iterator mit1=geo.begin();mit1!=geo.end();++mit1){
-                    const int orientation=geo.oriented(*mit0,*mit1);
-                    if (orientation!=0){
-                        operatorS(*mit1,*mit0,transmat,geo.sigma_inv(*mit0,*mit1)*(-1.0*K*orientation),gauss_order);
-                        operatorD(*mit1,*mit0,transmat,(K*orientation),gauss_order,true);
+                for (Geometry::const_iterator mit1 = geo.begin(); mit1 != geo.end(); ++mit1) {
```

Comment:
we need to setup a linter for these style changes otherwise we'll keep arguing with @papadop 

personally I prefer with space. Let's just have robot to tell us what style to adopt.